app/validators/return-requirements/start-date.validator.js

```diff
@@ -62,12 +62,12 @@ function _validateAnotherStartDate (payload, licenceStartDate, licenceEndDate) {
       .date()
       .format(['YYYY-MM-DD'])
       .required()
-      .greater(licenceStartDate)
+      .min(licenceStartDate)
       .less(licenceEndDate || '9999-12-31')
       .messages({
         'date.base': 'Enter a real start date',
         'date.format': 'Enter a real start date',
-        'date.greater': 'Start date must be after the original licence start date',
+        'date.min': 'Start date must be after the original licence start date',
```

Comment:
I know it is not in the ticket. But could we also update the error message to reflect the change.

```suggestion
        'date.min': 'Start date must be on or after the original licence start date',
```

Reply:
I knew I should've just made that change. Note to self: next time just do the common sense thing 🗒️ 🖊️